crates/bevy_ecs/src/query/mod.rs

```diff
@@ -21,6 +21,12 @@ pub use par_iter::*;
 pub use state::*;
 pub use world_query::*;
 
+use crate as bevy_ecs;
+
+/// A marker component that excludes an entity from queries that don't specifically request them.
```

Comment:
These docs need to be extended.

I would start with a doc test showing off how this differs from an ordinary component.